substrate/runtime/session/src/lib.rs

```diff
@@ -142,6 +142,11 @@ impl<T: Trait> Module<T> {
 	pub fn force_new_session(normal_rotation: bool) -> Result {
 		<ForcingNewSession<T>>::put(normal_rotation);
 		Ok(())
+	}	
+
+	/// Sets bad validators.
+	pub fn set_bad_validators(bad_validators: Vec<T::AccountId>) {
```

Comment:
any reason for this?

Reply:
used it in a past commit to set bad validators for early era slashing, not needed now after last update, will remove it.